docs/ref/forms/fields.txt

```diff
@@ -418,6 +418,7 @@ For each field, we describe the default widget used if you don't specify
         model field. See the :ref:`model field reference documentation on
         choices <field-choices>` for more details. If the argument is a
         callable, it is evaluated each time the field's form is initialized.
+        Defaults to the emtpy tuple.
```

Comment:
Maybe it's better to say, "an empty list" since that's what the attribute will be.